### DISCUSSION TITLE
Need help deciphering older version config code

### DISCUSSION THREAD

**Discussion by dbraun-ascendium** [11ty/eleventy#2646] [Q&A] [answered]
Hi, I'm trying to learn from someone's repo, and they have the .eleventy.js config like this: 
`module.exports = function (eleventyConfig) {
  return {
    dir: {
      input: "src",
      output: "dist",
    },
    templateFormats: ["html", "md", "njk"],
    passthroughFileCopy: true,
  };
};
`
It would seem like this is old code and the equivalent would be changing props on the eleventyConfig argument. But what isn't obvious to me is why passthoughFileCopy is being set to true. Would would be the update to this, and what does "true" do?

**Comment by pdehaan** [comment, marked as answer]
That's kind of interesting...

I've never used that config, but I did find mentions of `passthroughFileCopy` in _older_ versions of the docs; see https://v0-12-1.11ty.dev/docs/copy/#disabling-passthrough-file-copy. It seems to have been added in Jan 16, 2018 via https://github.com/11ty/eleventy/commit/904234193766aa9d9a7fb1e5037dea8838995a0f:
https://github.com/11ty/eleventy/search?q=passthroughFileCopy&type=commits

> | Configuration Option Key | Default Option | Valid Options | Command Line Override | Description |
> |----|----|----|----|----|
> | `passthroughFileCopy` | `true` | `true` or `false` | N/A | Files found (that aren’t templates) from white-listed file extensions will pass-through to the output directory. [Read more about Pass-through Copy](docs/copy.md). |

The `passthroughFileCopy:true` seems to be the default value in the older, pre-1.0 configs, like https://github.com/11ty/eleventy/blob/v0.11.x/src/defaultConfig.js#L39; and was initially `false` briefly but changed to `true` in https://github.com/11ty/eleventy/commit/90c43df9e05d09a0af09685cc45e394c0c2e1f21.

So I'm not sure why you'd need to set it explicitly to `true` since I assume in most/all cases that would have been default/unnecessary.
And the latest version of the passthrough docs is https://www.11ty.dev/docs/copy/ which shows the latest and greatest way to handle passthrough copying.
